packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -16,6 +16,7 @@ export const {
   debugRenderPhaseSideEffects,
   debugRenderPhaseSideEffectsForStrictMode,
   enableGetDerivedStateFromCatch,
+  enableSuspenseServerRenderer,
```

Comment:
Normally we would `export const enableSuspenseServerRenderer = true;` instead of this line. That way it should be enabled in www client but I don't know how we plan on using this file on the server in www. Perhaps we should configure the server build to just inline the feature flags in the build instead? Since this one is most meant for experiments based on GKs.

Reply:
I was planning on controlling this in www like so: https://fburl.com/diff/i1l0949n. 

let me know if you'd rather just hard-code the value here.